app/ami.py

```diff
@@ -143,6 +144,10 @@ def is_safe_to_terminate(self, today_date):
         resource_type = Ami
         return self.generic_is_safe_to_terminate(self, resource_type, today_date)
 
+    # Check if a instance is active
+    def is_active(self):
+        return True if self.state == 'available' else False
```

Comment:
Can be simplified to 

```suggestion
        return self.state == 'available'
```